rllib/utils/test_utils.py

```diff
@@ -510,7 +510,11 @@ def check_train_results(train_results):
         if "td_error" in policy_stats:
             configured_b = train_results["config"]["train_batch_size"]
             actual_b = policy_stats["td_error"].shape[0]
-            assert (configured_b - actual_b) / actual_b <= 0.1
+            # R2D2 case.
+            if (configured_b - actual_b) / actual_b > 0.1:
+                assert configured_b / (
+                    train_results["config"]["model"]["max_seq_len"] +
+                    train_results["config"]["burn_in"]) == actual_b
```

Comment:
maybe give this a bit of margin +-10% or something, float comparison isn't always reliable.

Reply:
try using np.isclose?